content/sensu-core/0.29/guides/intro-to-filters.md

```diff
@@ -98,6 +100,9 @@ in the handler definition. For example:
 
 To specify multiple Sensu event filters, use the `filters` attribute (plural).
 
+_WARNING: `filter` attribute will be depreciated in future releases. Please use 
```

Comment:
@portertech per https://docs.sensu.io/sensu-core/2.0/reference/handlers/#handler-attributes it looks like `filter` singular form is not supported in 2.0. Can you confirm?